src/_help/continuous-integration/cli.md

```diff
@@ -5,11 +5,11 @@ title: Bump CLI
 - TOC
 {:toc}
 
-The Bump.sh CLI is used to interact with your API documentation or hubs hosted on Bump.sh. With any API definition of your choice (from Swagger, OpenAPI or AsyncAPI), it can help you to:
+The Bump.sh CLI is used to interact with your API documentation or hubs hosted on Bump.sh. With any API definition of your choice (from OpenAPI, Swagger, or AsyncAPI), it can help you to:
 
-- Validate an API document before publishing to your documentation
-- Publish an API document to your Bump.sh documentation or hubs
-- Compare two API documents to generate a human-readable diff from your API definitions
+- Validate an API document before publishing to your documentation.
+- Publish an API document to your Bump.sh documentation or hubs.
+- Compare two API documents to generate a human-readable diff from your API description.
```

Comment:
We'll want to stay consistent with our glossary, and stick to "API definition" here, unless I misunderstand: https://docs.bump.sh/help/getting-started/concepts/#api-definition

Reply:
@ChristopheDujarric done